modules/monitoring-creating-cluster-monitoring-configmap.adoc

```diff
@@ -20,19 +20,7 @@ To configure the Prometheus Cluster Monitoring stack, you must create the cluste
 $ oc -n openshift-monitoring get configmap cluster-monitoring-config
 ----
 
-. If it does not exist, create it:
-+
-----
-$ oc -n openshift-monitoring create configmap cluster-monitoring-config
-----
-
-. Start editing the `cluster-monitoring-config` ConfigMap:
-+
-----
-$ oc -n openshift-monitoring edit configmap cluster-monitoring-config
-----
-
-. Create the `data` section if it does not exist yet:
+. If it does not exist, create one by applying the following manifest:
```

Comment:
I would suggest breaking this step into two parts so that we provide complete instructions on how to apply the manifest. For example:

```
. If it does not exist:
.. Create the following YAML manifest. In this example the file is called `cluster-monitoring-config.yaml`:
+
[source,yaml]
----
apiVersion: v1
kind: ConfigMap
metadata:
  name: cluster-monitoring-config
  namespace: openshift-monitoring
data:
  config.yaml: |
----
+
.. Then apply the configuration to create the ConfigMap:
+
[source,terminal]
----
$ oc apply -f cluster-monitoring-config.yaml
----
```

Reply:
great suggestion

Reply:
@olga-mir I have done the same in my restructuring PR (which is not ready yet to be re-reviewed.

You can see a full example of this suggested change here: https://github.com/openshift/openshift-docs/pull/24722/files#diff-3933f86a3f0ce5df04f6dac26dbbccc8

Reply:
Sorry I meant here: https://github.com/openshift/openshift-docs/pull/24722/files#diff-89302581129a792feee2baace787a6db

Reply:
Please feel free to reuse that example if you would like. Once you merge your PR, if there are any conflicts when I rebase my PR, I will resolve them. That restructuring PR is for master and 4.6 but will not be merged for a while yet as it will need an extensive review.